kubernetes/annotations_and_labels.md

```diff
@@ -8,6 +8,9 @@ This page defines common annotations and labels we set in Kubernetes objects.
   location. For now this should be in our `giantswarm/giantswarm` repository in
   which we crosslink all pages to create a reasonable documentation of all kinds
   of Kubernetes resources, their functionality and relationships.
+- `machine-deployment.giantswarm.io/subnet` - value should contain a subnet
```

Comment:
This is managed by `aws-operator`.

Reply:
Just for the record, the description describes the desired usage. The current implementation tracks the whole cluster CIDR for transitioning reasons.

Reply:
True, also noted that this is not a place to use `should`... :roll_eyes: